docs/core/compatibility/sdk/8.0/dotnet-restore-audit.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Breaking change: 'dotnet restore' produces security vulnerability warnings"
+description: Learn about a breaking change in the .NET 8 SDK where 'dotnet restore' produces security vulnerability warnings by default.
+ms.date: 08/15/2023
+---
+# 'dotnet restore' produces security vulnerability warnings
+
+The [`dotnet restore` command](../../../tools/dotnet-restore.md), which restores the dependencies and tools of a project, now produces security vulnerability warnings by default.
+
+## Previous behavior
+
+Previously, `dotnet restore` did not emit any security vulnerability warnings by default.
+
+## New behavior
+
+If you're developing with the .NET 8 SDK or a later version, `dotnet restore` produces security vulnerability warnings by default for *all* restored projects. When you load a solution, project, or run a CI/CD script, this change may break your workflow if you have `<TreatWarningsAsErrors>` enabled.
+
+## Version introduced
+
+.NET 8 Preview 4
+
+## Type of breaking change
+
+This change is a [behavioral change](../../categories.md#behavioral-change).
+
+## Reason for change
+
+In most cases when you restore a package, you want to know whether the restored package version contains any known security vulnerabilities. This functionality was added in order
```

Comment:
this seems incomplete.

Reply:
Sorry, definitely got distracted. Fixed for now, Re-review?